docs/usage/ComponentOverwrites.md

```diff
@@ -0,0 +1,28 @@
+# Component Overwrites
+
+When deploying applications using aggregated blueprints, their component descriptors reference multiple other components with blueprints.
+In order to test new versions of transitive dependencies one would have to release new versions of all including component descriptors which could, depending on the depth, result in a lot of releases and therefore repetitive overhead.
+
+In order to support simpler testing of development versions in aggregated components it should be possible to override specific blueprints or components manually in a livee system.
```

Comment:
Repetion of 'in order to' here. I suggest to just reduce the second one to just 'to': 'To support simpler testing ...'